app/models/widget/card.rb

```diff
@@ -10,12 +10,13 @@ class Widget::Card < ApplicationRecord
 
   validates_translation :title, presence: true
   validates :link_url, presence: true, if: -> { !header? || link_text.present? }
+  validates :order, numericality: { greater_than_or_equal_to: 1 }
 
   def self.header
     where(header: true)
   end
 
   def self.body
-    where(header: false, cardable_id: nil).order(:created_at)
+    where(header: false, cardable_id: nil).order(:order, :created_at)
```

Comment:
Since we're using `.order(:order, :created_at)` three times, do you think it makes sense to extract this logic to a scope?

```suggestion
    where(header: false, cardable_id: nil).by_order # or by_position, or similar
```